src/main/java/cheese/TaskList.java

```diff
@@ -0,0 +1,36 @@
+package cheese;
+
+import java.util.ArrayList;
+
+import cheese.task.Task;
+
+/**
+ * Stores the list of tasks for the chat bot
+ * All operations are the same as ArrayList
+ */
+public class TaskList {
+    private final ArrayList<Task> tasks;
+
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void add(Task task) {
+        tasks.add(task);
+    }
+
+    public int size() {
```

Comment:
Shouldn't the method name be a verb and not a noun? For example, getSize instead of size